ssl/record/ssl3_record.c

```diff
@@ -341,6 +341,10 @@ int ssl3_get_record(SSL *s)
                         if (strncmp((char *)p, "GET ", 4) == 0 ||
                             strncmp((char *)p, "POST ", 5) == 0 ||
                             strncmp((char *)p, "HEAD ", 5) == 0 ||
+                            strncmp((char *)p, "PATCH", 5) == 0 ||
+                            strncmp((char *)p, "OPTIO", 5) == 0 ||
```

Comment:
Why do these check for partial words? As opposed to `PATCH `, `OPTION ` etc?  Just curious.